### PR TITLE
storaged: Provide default names for volume groups and logical volumes

### LINKED ISSUE #3626: storage: suggest name for a new lvm volume
Similar to how we do with new network connections like bonds (bond0), vlans (vlan0) etc.
Makes it quicker to go through the dialog if you actually don't really care about a specific name that much.

### PR DESCRIPTION
**Pull request by mvollmer**
Fixes #3626

### CODE REVIEW

File: pkg/storaged/details.js [hunk 1]
```diff
@@ -764,10 +764,26 @@ define([
                 if (!vgroup)
                     return;
 
+                function find_lvol(name) {
+                    var lvols = client.vgroups_lvols[path];
+                    for (var i = 0; i < lvols.length; i++)
```
Comment (stefwalter) [inline pkg/storaged/details.js]: Even though they're not strictly necessary, it's clearer when we use braces around multiline blocks.

**Comment by stefwalter** [comment]
This needs an integration test (or a unit test). Probably a one line change to an already present integration test would suffice.

File: pkg/storaged/details.js (hunk 1 above)
Comment (mvollmer) [inline pkg/storaged/details.js]: Yes, fixed.

**Comment by mvollmer** [comment]
> This needs an integration test (or a unit test).

Done.

**Pull request closed without merging**